build/app_template_files/__NativeSwiftTemplateAppName__/__NativeSwiftTemplateAppName__/SalesforceSDKAssets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The template apps no longer need any control over SalesforceSDKAssets.  That asset catalog should be delivered via the artifacts of the pod install.